examples/bri-3/package.json

```diff
@@ -31,11 +31,19 @@
     "@nestjs/common": "^9.0.0",
     "@nestjs/core": "^9.0.0",
     "@nestjs/cqrs": "^9.0.1",
+    "@nestjs/jwt": "^9.0.0",
```

Comment:
still reviewing but 10 more dependencies for this seems a lot, why we need passport and nest jwt?

Reply:
kindly skim read here: https://docs.nestjs.com/security/authentication#authentication